pkg/kubelet/kubelet.go

```diff
@@ -1124,16 +1124,25 @@ func initializeServerCertificateManager(kubeClient clientset.Interface, kubeCfg
 		CertificateSigningRequestClient: certSigningRequestClient,
 		Template: &x509.CertificateRequest{
 			Subject: pkix.Name{
-				CommonName:   string(nodeName),
+				CommonName:   fmt.Sprintf("system:node:%s", nodeName),
 				Organization: []string{"system:nodes"},
 			},
 			DNSNames:    hostnames,
 			IPAddresses: ips,
 		},
 		Usages: []certificates.KeyUsage{
+			// https://tools.ietf.org/html/rfc5280#section-4.2.1.3
+			//
+			// Signing allows the certificate to be used to verify
+			// digital signatures used during TLS negotiation.
+			certificates.UsageSigning,
```

Comment:
can we actually change the kubelet to use the usage that corresponds to the RFC? "signing" is pretty ambiguous